.github/workflows/orca.yml

```
@@ -32,23 +32,23 @@ jobs:
       matrix:
         orca-job:
           - STATIC_CODE_ANALYSIS # Static code analysis.
-          - INTEGRATED_TEST_ON_OLDEST_SUPPORTED # Integrated test on oldest supported Drupal core version.
-          - INTEGRATED_TEST_ON_LATEST_LTS # Integrated test on latest LTS Drupal core version.
-          - INTEGRATED_TEST_ON_PREVIOUS_MINOR # Integrated test on previous minor Drupal core version.
-          - INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR # Integrated upgrade test from previous minor.
-          - ISOLATED_TEST_ON_CURRENT # Isolated test on current Drupal core version.
+          # - INTEGRATED_TEST_ON_OLDEST_SUPPORTED # Integrated test on oldest supported Drupal core version.
+          # - INTEGRATED_TEST_ON_LATEST_LTS # Integrated test on latest LTS Drupal core version.
+          # - INTEGRATED_TEST_ON_PREVIOUS_MINOR # Integrated test on previous minor Drupal core version.
+          # - INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR # Integrated upgrade test from previous minor.
+          # - ISOLATED_TEST_ON_CURRENT # Isolated test on current Drupal core version.
           # This job customized below under "include".
           # - INTEGRATED_TEST_ON_CURRENT # Integrated test on current Drupal core version.
-          - ISOLATED_TEST_ON_CURRENT_DEV # Isolated test on current dev Drupal core version.
-          - INTEGRATED_TEST_ON_CURRENT_DEV # Integrated test on current dev Drupal core version.
-          - LOOSE_DEPRECATED_CODE_SCAN # Loose deprecated code scan.
-          - STRICT_DEPRECATED_CODE_SCAN # Strict deprecated code scan.
-          - DEPRECATED_CODE_SCAN_W_CONTRIB # Deprecated code scan w/ contrib.
+          # - ISOLATED_TEST_ON_CURRENT_DEV # Isolated test on current dev Drupal core version.
+          # - INTEGRATED_TEST_ON_CURRENT_DEV # Integrated test on current dev Drupal core version.
+          # - LOOSE_DEPRECATED_CODE_SCAN # Loose deprecated code scan.
+          # - STRICT_DEPRECATED_CODE_SCAN # Strict deprecated code scan.
+          # - DEPRECATED_CODE_SCAN_W_CONTRIB # Deprecated code scan w/ contrib.
           - ISOLATED_TEST_ON_NEXT_MINOR # Isolated test on next minor Drupal core version.
           - INTEGRATED_TEST_ON_NEXT_MINOR # Integrated test on next minor Drupal core version.
           - ISOLATED_TEST_ON_NEXT_MINOR_DEV # Isolated test on next minor dev Drupal core version.
           - INTEGRATED_TEST_ON_NEXT_MINOR_DEV # Integrated test on next minor dev Drupal core version.
```

These need to be commented out, too, because they still resolve to Drupal 9.x. The `*_TEST_ON_NEXT_MAJOR_*` jobs down in `jobs.build.strategy.matrix.orca-job` are the ones that matter right now (and they're already correct).

```suggestion
         # - ISOLATED_TEST_ON_NEXT_MINOR # Isolated test on next minor Drupal core version.
         # - INTEGRATED_TEST_ON_NEXT_MINOR # Integrated test on next minor Drupal core version.
         # - ISOLATED_TEST_ON_NEXT_MINOR_DEV # Isolated test on next minor dev Drupal core version.
         # - INTEGRATED_TEST_ON_NEXT_MINOR_DEV # Integrated test on next minor dev Drupal core version.
```